test/local/senders/sms.js

```diff
@@ -250,7 +195,6 @@ describe('lib/senders/sms:', () => {
         sms: {
           apiKey: 'foo',
           apiSecret: 'bar',
-          balanceThreshold: 1,
```

Comment:
Could probably remove it from [config](https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L711) too.